dataland-backend/backendOpenApi.json

```diff
@@ -2847,6 +2847,16 @@
         }
       },
       "LksgGeneral": {
+        "required": [
+          "dataDate",
+          "listOfProductionSites",
```

Comment:
There's a white margin around the uplaod form that shouldn't be there...
![image](https://user-images.githubusercontent.com/47553592/230322782-67c5c7ad-9e3b-4d6a-8bd3-e2a6722c8def.png)

Reply:
Fixed for other pages (e.g. Choose company page, lksg view page) by setting padding of the `TheContent` slot to `p-0`.

Reply:
ok